app/components/institution-page/campus-program-application-modal.ts

```diff
@@ -16,6 +18,8 @@ interface Signature {
 
 export default class CampusProgramApplicationModal extends Component<Signature> {
   @service declare authenticator: AuthenticatorService;
+  @tracked emailAddressToPrefill: string | undefined = undefined;
```

Comment:
We usually have a space between `@service ..` declarations and `@tracked ...` declarations

<img width="770" height="822" alt="Screenshot 2025-08-16 at 02 46 11" src="https://github.com/user-attachments/assets/9e3e837f-801f-4189-854a-aca99db94343" />